docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM alpine:latest as build
+
+ARG PKG_MANAGER="yarn"
+ARG INSTALL_COMMAND="yarn install --pure-lockfile --production"
+
+RUN mkdir /opt/armadietto
+WORKDIR /opt/armadietto
+
+RUN apk add nodejs $PKG_MANAGER
+
+COPY package.json ./
+COPY package-lock.json ./
+COPY yarn.lock ./
+
+RUN $INSTALL_COMMAND
+
+FROM alpine:latest
+
+LABEL description="Armadietto Node.js web service (a remoteStorage server)"
+
+ARG CONFIG_PATH_STORAGE="/usr/share/armadietto"
+ARG PROJECT_NAME="armadietto"
+ARG PORT="8000"
+ARG USER="armadietto"
+
+ENV NODE_ENV=production
+ENV PROJECT_NAME=$PROJECT_NAME
+ENV PORT=$PORT
+
+RUN mkdir /opt/armadietto
+WORKDIR /opt/armadietto
+
+RUN apk add nodejs
```

Comment:
Is it OK to always bring in latest version?  Should this be explicitly controlled?  I guess same for `alpine:latest` on line 1.

I'm cool with it as it is, just thinking out loud, in light of recent debacles.

Was also for indeterministic bits between October and now that could cause my failures building.  But I don't think this is it either.

Reply:
Not sure what the default tag there points to, but latest active LTS makes most sense to me for node.js.